tests/test_lambdas.py

```diff
@@ -3,7 +3,7 @@
 import lambdas
 
 
-def test_lambda_handler_success(lambda_function_event_payload):
+def test_lambda_handler_success(lambda_function_event_payload, mock_parse_oidc_data):
```

Comment:
Nice!  Mocking that utility function feels like a handy way to simulate different data coming from the header JWT payloads, without mocking them themselves.